lib/seed_dump/environment.rb

```diff
@@ -9,21 +8,22 @@ def dump_using_environment(env = {})
                else
                  ActiveRecord::Base.descendants.select do |model|
                    (model.to_s != 'ActiveRecord::SchemaMigration') && \
+                   (model.to_s != 'ActiveRecord::InternalMetadata') && \
```

Comment:
I'm not sure if this was working before the ruby upgrade or not, but this appears to be an ActiveRecord.Base descendant that we want to ignore along with schema migrations.